app/models/publisher.rb

```diff
@@ -150,6 +150,21 @@ def email_verified?
     email.present?
   end
 
+  # Public: Show history of publisher's notes and statuses sorted by the created time
+  #
+  # Returns an array of PublisherNote and PublisherStatusUpdate
+  def history
+    # Create hash with created_at time as the key
+    # Then we can merge and sort by the key to get history
+    notes = self.notes.map { |n| { n.created_at => n} }
+    status = self.status_updates.map { |s| { s.created_at => s } }
+
+    combined = notes + status
+    combined = combined.sort { |x, y| x.keys.first <=> y.keys.first}.reverse
```

Comment:
If the key is in string notation for datetime, wouldn't you have to cast this the string?

Reply:
Actually Ruby allows for any object to be the key so fortunately this is in the right type when it's being sorted. 

```
[4] pry(#<Publisher>)> combined[0].keys[0].class
=> ActiveSupport::TimeWithZone
```